src/lasp_type.erl

```diff
@@ -101,15 +102,24 @@ new(Type) ->
 update(Type, Operation, Actor, Value) ->
     Mode = get_mode(),
     T = get_type(remove_args(Type), Mode),
+    RealActor = get_actor(T, Actor),
     case Mode of
         delta_based ->
-            T:delta_mutate(Operation, Actor, Value);
+            T:delta_mutate(Operation, RealActor, Value);
         state_based ->
-            T:mutate(Operation, Actor, Value);
+            T:mutate(Operation, RealActor, Value);
         pure_op_based ->
             ok %% @todo
     end.
 
+%%private
+get_actor(state_oorset_ps, {{StorageId, _TypeId}, Actor}) ->
```

Comment:
Given that the optimized representations require an actor that's unique for a given object, like `{ObjectId, ActorId}`, I was thinking we could just standardize this in the backend for everything.